docs/mindsdb-docs/docs/sql/tutorials/customer-churn.md

```diff
@@ -1,127 +1,196 @@
+
+!!!Info "Labels and Features"
+
+    A **label** is the thing we're predicting — the y variable in simple linear regression.
+    A **feature** is an input variable — the x variable in simple linear regression.
+
+## Training a Predictor Via [`#!sql CREATE PREDICTOR`](/sql/create/predictor)
+
+Let's create and train your first machine learning predictor. For that, we are going to use the [`#!sql CREATE PREDICTOR`](/sql/create/predictor) syntax where we specify what sub-query to train `#!sql FROM` (features) and what we want to `#!sql PREDICT` (labels).
 
 ```sql
-CREATE PREDICTOR predictor_name
+CREATE PREDICTOR mindsdb.customer_churn_predictor
 FROM files
-  (SELECT * FROM file_name)
-PREDICT column_name;
+  (SELECT * FROM churn)
+PREDICT Churn;
 ```
 
-The required values that we need to provide are:
+We use all of the columns as features, except for the `Churn` column whose value is going to be predicted.
 
-- predictor_name (string) - The name of the model
-- file_name - The name of the table/file you have uploaded.
-- column_name (string) - The feature you want to predict.
+## Checking the Status of a Predictor
 
-To train the model that will predict customer churn, run the following syntax:
+A predictor may take a couple of minutes for the training to complete. You can monitor the status of your predictor by using this SQL command:
 
 ```sql
-CREATE PREDICTOR customer_churn
-FROM files
-  (SELECT * FROM churn)
-PREDICT Churn;
+SELECT status
+FROM mindsdb.predictors
+WHERE name='customer_churn_predictor';
 ```
 
-![Create predictor](/assets/sql/tutorials/customer_churn/create_churn.png)
+If we run it right after creating a predictor, we'll most probably get this output:
 
-What we did here was to create a predictor called customer_churn to predict the Churn and also ignore the gender column as an irrelevant column for the model.The model training has started. 
+```sql
++----------+
+| status   |
++----------+
+| training |
++----------+
+```
 
-You can check the predictors status and will be able to make a prediction once the status shows complete:
+But if we wait a couple of minutes, this should be the output:
 
 ```sql
-SELECT * FROM mindsdb.predictors where name='customer_churn';
++----------+
+| status   |
++----------+
+| complete |
++----------+
 ```
 
-![select](/assets/sql/tutorials/customer_churn/select.png)
+Now, if the status of our predictor says `complete`, we can start making predictions!
+
+## Making Predictions
+
+You can make predictions by querying the predictor as if it were a table. The [`SELECT`](/sql/api/select/) syntax lets you make predictions for the label based on the chosen features.
+
+```sql
+SELECT Churn, Churn_confidence, Churn_explain
+FROM mindsdb.customer_churn_predictor
+WHERE SeniorCitizen=0 AND Partner='Yes' AND Dependents='No' AND tenure=1 AND PhoneService='No' AND MultipleLines='No phone service' AND InternetService='DSL';
```

Comment:
Break into new lines each AND for readability.

Reply:
Done.